### DISCUSSION TITLE
Is there a way to get the PHP version set for each project to reflect on the terminal?

### DISCUSSION THREAD

**Discussion by frankliniwobi** [beyondcode/herd-community#678] [Q&A] [answered]
In Herd you can set different PHP versions for different projects, however, the PHP version that is used on the terminal is the 'Global PHP Version' set in the herd dashboard irrespective of your current directory.

For example, I have 3 projects

blog.test - using PHP 8.3
test.test - using PHP 7.4
saas.test - using PHP 8.1

Even after setting this manually on the Sites page of herd, when I `cd` into the test.test directory and run `php -v` I still get 8.3 as the PHP version instead of 7.4 because 8.3 is the 'Global PHP Version' I set in the herd dashboard, is there a way to fix this?

**Comment by mpociot** [comment, marked as answer]
Use "herd php" and "herd composer" inside a project directory to use the isolated version of PHP.

**Reply by frankliniwobi** [reply]
Oh! Thanks 🙏
